etc/config/kotlin.defaults.properties

```
@@ -0,0 +1,18 @@
+# Default settings for Kotlin/JVM
+compilers=&kotlin
+compilerType=kotlin
+versionFlag=-version
+objdumper=javap
+instructionSet=java
+
+group.kotlin.compilers=kotlincdefault
+compiler.kotlincdefault.exe=/usr/bin/kotlinc
```

We are not interested in allowing kotlinc-js to be invoked. The kotlinc-jvm driver should be used for this.

```suggestion
compiler.kotlincdefault.exe=/usr/bin/kotlinc-jvm
```